### PR TITLE
Update containerd

### PR DESCRIPTION
**Pull request by Random-Liu**
Fixes https://github.com/containerd/cri/issues/762.
Fixes https://github.com/containerd/cri/issues/772.

Update containerd once to fix https://github.com/containerd/cri/issues/772 and https://github.com/containerd/cri/issues/762.

### CODE REVIEW

File: vendor.conf [hunk 1]
```diff
@@ -4,11 +4,11 @@ github.com/boltdb/bolt e9cf4fae01b5a8ff89d0ec6b32f0d9c9f79aefdd
 github.com/BurntSushi/toml a368813c5e648fee92e5f6c30e3944ff9d5e8895
 github.com/containerd/cgroups fe281dd265766145e943a034aa41086474ea6130
 github.com/containerd/console cb7008ab3d8359b78c5f464cb7cf160107ad5925
-github.com/containerd/containerd 1381f8fddc4f826e12b48d46c9def347d5aa338a
+github.com/containerd/containerd d1435e6e4dcffd99e0da396ff771b5bbe0d93f5e
```
Comment (mikebrow) [inline vendor.conf]: containerd vendor update looks good..

File: cluster/gce/configure.sh [hunk 2]
```diff
@@ -155,6 +155,8 @@ if [ -n "${network_policy_provider}" ] && [ "${network_policy_provider}" != "non
 fi
 log_level="${CONTAINERD_LOG_LEVEL:-"info"}"
 cat > ${config_path} <<EOF
+disabled_plugins = ["restart"]
```
Comment (mikebrow) [inline cluster/gce/configure.sh]: nit.. can we explain or link to the reason for disabling restart here... for example:
```# containerd provides it's own restart manager```

File: pkg/server/image_pull.go [hunk 3]
```diff
@@ -100,22 +100,16 @@ func (c *criService) PullImage(ctx context.Context, r *runtime.PullImageRequest)
...
 
-	// TODO(mikebrow): add truncIndex for image id
 	image, err := c.client.Pull(ctx, ref,
 		containerd.WithSchema1Conversion,
 		containerd.WithResolver(resolver),
+		containerd.WithPullSnapshotter(c.config.ContainerdConfig.Snapshotter),
+		containerd.WithPullUnpack,
 	)
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to pull image %q", ref)
```
Comment (mikebrow) [inline pkg/server/image_pull.go]: failed to pull and unpack image %q

File: pkg/server/image_pull.go [hunk 4]
```diff
@@ -100,22 +100,16 @@ func (c *criService) PullImage(ctx context.Context, r *runtime.PullImageRequest)
 	// image has already been converted.
 	isSchema1 := desc.MediaType == containerdimages.MediaTypeDockerSchema1Manifest
 
-	// TODO(mikebrow): add truncIndex for image id
```
Comment (mikebrow) [inline pkg/server/image_pull.go]: might want to keep the todo for truncated image names..

File: cluster/gce/configure.sh (hunk 2 above)
Comment (Random-Liu) [reply cluster/gce/configure.sh]: Will do.

File: pkg/server/image_pull.go (hunk 4 above)
Comment (Random-Liu) [reply pkg/server/image_pull.go]: We already supported it, right?

File: cluster/gce/configure.sh (hunk 2 above)
Comment (Random-Liu) [reply cluster/gce/configure.sh]: Done

File: pkg/server/image_pull.go (hunk 3 above)
Comment (Random-Liu) [reply pkg/server/image_pull.go]: Done.

File: pkg/server/image_pull.go (hunk 4 above)
Comment (Random-Liu) [reply pkg/server/image_pull.go]: https://github.com/containerd/cri/pull/375

Comment (mikebrow) [reply pkg/server/image_pull.go]: oops :-)

**Pull request merged**